packages/phrases/src/locales/it/translation/admin-console/user-details.ts

```diff
@@ -2,7 +2,7 @@ const user_details = {
   page_title: 'Dettagli utente',
   back_to_users: 'Torna alla gestione utenti',
   created_title: 'Questo utente è stato creato con successo',
-  created_guide: 'Ecco le informazioni per aiutare l’utente con il processo di accesso.',
+  created_guide: "Ecco le informazioni per aiutare l'utente con il processo di accesso.",
```

Comment:
‘ => " by design?

Reply:
There is a single quote mark in the sentence, so use double quotes is expected.

Reply:
ok, just notice that.